test/testcases.py

```diff
@@ -70,29 +70,38 @@ def test_testcase_nameing():
     assert f"{tc}" == expected, f"{tc} != {expected}"
 
 
+@dataclasses.dataclass
+class TestCaseC10S(TestCase):
+    container_ref: str = os.getenv(
+        "BIB_TEST_BOOTC_CONTAINER_TAG",
+        "quay.io/centos-bootc/centos-bootc:stream10")
+    osinfo_template: str = "CentOS Stream 10 ({arch})"
```

Comment:
Ups, I overlooked this, this will probably need to be tweaked slightly as this code has changed: 
[0001-test-tweak-TestCaseC10S-testcase-and-osinfo_for-help.patch.txt](https://github.com/user-attachments/files/17951111/0001-test-tweak-TestCaseC10S-testcase-and-osinfo_for-help.patch.txt)

Reply:
We should probably refactor this again :/ it would be nice if osinfo would life closer to the testcases, they did originally belong together but then the test-case naming became too verbose, but the current split (which is my fault) is also not ideal